README.md

```diff
@@ -67,10 +67,9 @@ _**Required**_
 * [LogDNA Ingestion Key](https://app.logdna.com/manage/profile)
 
 _**Optional**_
-* Hostname - ([string][]) - max length 32 chars
+* Hostname - ([string][])
 * MAC Address - ([string][])
```

Comment:
@Samir, I have some general comments, on sections above (but of course GH doesn't let me comment on "old" sections):

1. line # 17: In the header let's use **fileConfig** as one word, like it is in the body content.
2. Generally, across the doc, let's use the angle brackets format for any variables, so for example <my sample log line> or <my custom app>.
3. line # 136 needs a period at end of sentence.

Reply:
Done, @tanberry!